common/src/main/java/com/revenuecat/purchases/common/errorLogUtils.kt

```diff
@@ -0,0 +1,101 @@
+package com.revenuecat.purchases.common
+
+import com.revenuecat.purchases.strings.Emojis
+
+fun googleErrorLog(level: LogLevel, message: String) {
+    val emojiList = listOf(Emojis.GOOGLE_ERROR, Emojis.DOUBLE_EXCLAMATION)
+    val emojifiedMessage = "${emojiList.joinToString("")} $message"
+    when (level) {
+        LogLevel.DEBUG -> debugLog(emojifiedMessage)
+        LogLevel.INFO -> log(emojifiedMessage)
+        LogLevel.ERROR -> errorLog(emojifiedMessage)
+    }
+}
+
+fun infoLog(level: LogLevel, message: String) {
+    val emojiList = listOf(Emojis.INFO)
+    val emojifiedMessage = "${emojiList.joinToString("")} $message"
+    when (level) {
+        LogLevel.DEBUG -> debugLog(emojifiedMessage)
+        LogLevel.INFO -> log(emojifiedMessage)
+        LogLevel.ERROR -> errorLog(emojifiedMessage)
+    }
+}
+
+fun purchaseLog(level: LogLevel, message: String) {
+    val emojiList = listOf(Emojis.PURCHASE)
+    val emojifiedMessage = "${emojiList.joinToString("")} $message"
+    when (level) {
+        LogLevel.DEBUG -> debugLog(emojifiedMessage)
+        LogLevel.INFO -> log(emojifiedMessage)
+        LogLevel.ERROR -> errorLog(emojifiedMessage)
+    }
+}
+
+fun rcSuccessLog(level: LogLevel, message: String) {
+    val emojiList = listOf(Emojis.RC_SUCCESS)
+    val emojifiedMessage = "${emojiList.joinToString("")} $message"
+    when (level) {
+        LogLevel.DEBUG -> debugLog(emojifiedMessage)
+        LogLevel.INFO -> log(emojifiedMessage)
+        LogLevel.ERROR -> errorLog(emojifiedMessage)
+    }
+}
+
+fun rcPurchaseSuccessLog(level: LogLevel, message: String) {
+    val emojiList = listOf(Emojis.RC_SUCCESS, Emojis.PURCHASE)
+    val emojifiedMessage = "${emojiList.joinToString("")} $message"
```

Comment:
the logic to add the emojis is also repeated quite a bit, we should try to unify it